### PR TITLE
Document how to create user or api key for standalone agent

### LINKED ISSUE #749: Document how to create user or API key for standalone Elastic Agent
How to create a user or API key with necessary permissions for agent in standalone mode in ES, and then configure that in the agent policy.

copied from https://github.com/elastic/obs-dc-team/issues/147

### PR DESCRIPTION
**Pull request by dedemorton**
Closes #749 

Preview link for new topic: https://observability-docs_1340.docs-preview.app.elstc.co/guide/en/fleet/master/grant-access-to-elasticsearch.html

### CODE REVIEW

**Comment by dedemorton** [comment]
@nimarezainia Who should review this? @ruflin provided some input, but I dunno if he has the bandwidth. Thanks!

**Comment by nimarezainia** [comment]
> @nimarezainia Who should review this? @ruflin provided some input, but I dunno if he has the bandwidth. Thanks!

@jlind23 could you recommend an engineering familiar with the code and behavior to review this?

File: docs/en/ingest-management/elastic-agent/grant-access-to-elasticsearch.asciidoc
Comment (ruflin) [inline docs/en/ingest-management/elastic-agent/grant-access-to-elasticsearch.asciidoc:9]: `logs-*-*` is required. I know we still use `logs-*` in many places and it works too but we likely to reduce it to just `logs-*-*` if possible. Same for the other patterns.

File: docs/en/ingest-management/elastic-agent/grant-access-to-elasticsearch.asciidoc
Comment (ruflin) [inline docs/en/ingest-management/elastic-agent/grant-access-to-elasticsearch.asciidoc:54]:
```suggestion
          "logs-*-*", "metrics-*-*", "traces-*-*", "synthetics-*-*"
```

File: docs/en/ingest-management/elastic-agent/grant-access-to-elasticsearch.asciidoc
Comment (ruflin) [inline docs/en/ingest-management/elastic-agent/grant-access-to-elasticsearch.asciidoc:63]: Can we add a note that in many scenarios not all these permissions are needed. For example if not apm or synthetics is used, only logs and metrics are necessary.

File: docs/en/ingest-management/elastic-agent/grant-access-to-elasticsearch.asciidoc
Comment (ruflin) [inline docs/en/ingest-management/elastic-agent/grant-access-to-elasticsearch.asciidoc:71]: @nimarezainia @mostlyjason This is interesting. Should we improve this?

File: docs/en/ingest-management/elastic-agent/grant-access-to-elasticsearch.asciidoc
Comment (ruflin) [inline docs/en/ingest-management/elastic-agent/grant-access-to-elasticsearch.asciidoc:119]: See above

**Comment by jlind23** [comment]
> @jlind23 could you recommend an engineering familiar with the code and behavior to review this?
@michalpristas may be the best here.
@michalpristas did you also start handing this over?

File: docs/en/ingest-management/elastic-agent/grant-access-to-elasticsearch.asciidoc
Comment (dedemorton) [reply docs/en/ingest-management/elastic-agent/grant-access-to-elasticsearch.asciidoc:71]: (I think we should!)

**Comment by dedemorton** [comment]
I've pinged Michal on slack, but if anyone else is ready to approve this, I can probably merge without his review.

File: docs/en/ingest-management/elastic-agent/grant-access-to-elasticsearch.asciidoc
Comment (ruflin) [inline docs/en/ingest-management/elastic-agent/grant-access-to-elasticsearch.asciidoc:109]: Note: This does NOT block this PR

@nimarezainia I wonder if we should even document this here as we don't recommend this approach.

**Review by ruflin — APPROVED** [review]

File: docs/en/ingest-management/elastic-agent/grant-access-to-elasticsearch.asciidoc
Comment (mostlyjason) [reply docs/en/ingest-management/elastic-agent/grant-access-to-elasticsearch.asciidoc:74]: We are proposing a way to automatically generate the API key here https://github.com/elastic/kibana/issues/80841

File: docs/en/ingest-management/elastic-agent/grant-access-to-elasticsearch.asciidoc
Comment (dedemorton) [reply docs/en/ingest-management/elastic-agent/grant-access-to-elasticsearch.asciidoc:109]: I think we should at least for now because the standalone policy that gets generated shows those settings.

**Pull request merged**
